Detectors/ZDC/simulation/src/Detector.cxx

```diff
@@ -2269,7 +2269,9 @@ void Detector::Register()
 //_____________________________________________________________________________
 void Detector::Reset()
 {
-  mHits->clear();
-  mLastPrincipalTrackEntered = -1;
-  resetHitIndices();
+  if (!o2::utils::ShmManager::Instance().isOperational()) {
```

Comment:
I think the protection should only be around `mHits->clear()` because the rest needs to be done irrespective of shared memory or not.

Reply:
right, done!